### ISSUE TITLE
Cap.so Raycast extension

### ISSUE THREAD

**Issue by simonayotte** [CapSoftware/Cap#28]
**Support Cap.so with an official Raycast extension.**

**Includes minimally** two commands that can be invoked from Raycast:
- Start recording (w/ default settings set by user)
- Stop recording (Copies the video URL by default at the end of the recording)

Ideally includes support for:
- Opening the web dashboard

**Comment by neo773** [comment]
Could I get assigned to this?
I’ve already authored 3 extensions on [Raycast](https://www.raycast.com/huzef44)

**Comment by hknhhy333** [comment]
good

**Comment by richiemcilroy** [comment]
This is currently in progress by @ItsEeleeya - almost finished getting it released!

**Comment by richiemcilroy** [comment]
Sorry for the slow progress here. Over the last couple of months we have completely redeveloped the app from the ground up, leading to the launch of Cap v0.3.0. It's a different architecture (Tauri v2 + SolidJS).

@ItsEeleeya - would you like to look at updating for the new version of the app?

**Comment by ItsEeleeya** [comment]
Definitely, I've only been watching the rapid re-make of the app, it's looking awesome btw!
I'm looking to add more features to this soon and then re-implement it for the app.
One of the features I've had in mind was connecting your cap account to the extension so you could view/edit/share any recordings or screenshots.
Raycast only supports PKCE for OAuth. I believe NextAuth supports this method but I might need a bit of help making extra sure this part is implemented correctly!

It think it should also allow you to only use the web recorder and trigger the start/stop via the extension.

I'll be mostly available by the end of the next weekend!

**Comment by raphaelbernhart** [comment]
Just checking in, is this still needed? :)

**Comment by iamsoum** [comment]
Is this still being worked on? I don't see any extension in Raycast and the this issue is not closed yet.

**Comment by ItsEeleeya** [comment]
@raphaelbernhart @iamsoum Yes, it's going to be updated soon for the new version of Cap
I've been working on the Windows compatibility recently and this will be my priority

**Comment by svliantiss** [comment]
Is this working yet, I'd love to use it :)

**Comment by JYZ-LESLIE** [comment]
/attempt #28

Scoped plan for current v0.3 architecture:
1. Add lightweight  and  deep links in desktop so external tools can trigger start/stop without requiring capture-mode payloads.
2. Make  start using the user’s existing recording defaults (target/mic/camera/mode/system-audio) from .
3. Add a minimal Raycast extension () with 3 no-view commands: Start Recording, Stop Recording, Open Dashboard.
4. Submit as a focused PR with  in the PR body.

**Comment by JYZ-LESLIE** [comment]
/attempt #28

Scoped plan for current v0.3 architecture:
1. Add lightweight `cap://record` and `cap://stop` deep links in desktop so external tools can trigger start/stop without requiring capture-mode payloads.
2. Make `cap://record` start using the user’s existing recording defaults (target/mic/camera/mode/system-audio) from `RecordingSettingsStore`.
3. Add a minimal Raycast extension (`extensions/raycast`) with 3 no-view commands: Start Recording, Stop Recording, Open Dashboard.
4. Submit as a focused PR with `/claim #28` in the PR body.

**Comment by JYZ-LESLIE** [comment]
Submitted scoped implementation PR: https://github.com/CapSoftware/Cap/pull/1830\n\nHighlights:\n-  and  deep links\n-  starts with user-saved defaults\n- new  commands for start/stop/dashboard\n

**Comment by JYZ-LESLIE** [comment]
Submitted scoped implementation PR: https://github.com/CapSoftware/Cap/pull/1830

Highlights:
- cap://record and cap://stop deep links
- cap://record starts with user-saved defaults
- new extensions/raycast commands for start/stop/dashboard

**Comment by JYZ-LESLIE** [comment]
/claim #28

PR: https://github.com/CapSoftware/Cap/pull/1830
